.circleci/config.yml

```diff
@@ -65,6 +73,10 @@ workflows:
         filters:
           branches:
             ignore: /pull\/[0-9]+/ # Ignore running on forked PRs
+      - e2e_test_as_cli:
+        filters:
+          branches:
+            ignore: /pull\/[0-9]+/ # Ignore running on forked PRs
```

Comment:
Are we sure this will work?

e.g. this branch's name is DXCDT-120-e2e-test-script-cli and doesn't start with /pull. That's the branch URL when coming from a fork.